CHANGELOG.md

```diff
@@ -47,6 +48,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Removed
 
+### Dependencies
+
+| Dependency             | Old version | New version |
+| ---------------------- | ----------- | ----------- | ----------------------------- |
+| `ascat`                | 3.0.0       | unchanged   |
+| `bcftools`             | 1.15.1      | unchanged   |
+| `bwa-mem2`             | 2.2.1       | unchanged   |
+| `bwa`                  | 0.7.17      | unchanged   |
+| `cancerit-allelecount` | 4.3.0       | unchanged   |
+| `cnvkit`               | 0.9.9       | unchanged   |
+| `control-freec`        | 11.6        | unchanged   |
+| `deepvariant`          | 1.3.0       | unchanged   |
+| `dragmap`              | 1.2.1       | unchanged   |
+| `ensembl-vep`          | 106.1       | unchanged   | //should we update this here? |
```

Comment:
Did I forgot to update that previously?

Reply:
think so, do you have time to update vep, I think they are at 108, or should we push it? (you should show me how I can do that myself :) )

Reply:
Updating module is easy, just the one thing we need to do on top of that is building the containers with cache